addons/l10n_in/models/res_config_settings.py

```diff
@@ -9,3 +9,5 @@ class ResConfigSettings(models.TransientModel):
 
     group_l10n_in_reseller = fields.Boolean(implied_group='l10n_in.group_l10n_in_reseller', string="Manage Reseller(E-Commerce)")
     module_l10n_in_edi = fields.Boolean('Indian Electronic Invoicing')
+    module_l10n_in_edi_ewaybill = fields.Boolean('Indian Electronic Waybill')
+    module_l10n_in_reports = fields.Boolean('Indian - Accounting Reports')
```

Comment:
```suggestion
```
The l10n_*_reports are auto install, so I don't think it's useful to have it in the config view :thinking: 
It will only not be installed if you don't have account_accountant, in which case allowing to install an enterprise module via a setting in a community module may not be the best idea.

Reply:
okay